op-acceptance-tests/tests/interop/reorgs/interop_reorg_test.go

```diff
@@ -0,0 +1,209 @@
+package reorgs
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/devstack/devtest"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/devstack/presets"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/devstack/stack/match"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/txplan"
+	"github.com/ethereum-optimism/optimism/op-test-sequencer/sequencer/seqtypes"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+var SimpleInterop presets.TestSetup[*presets.SimpleInterop]
+
+// TestMain creates the test-setups against the shared backend
+func TestMain(m *testing.M) {
+	// Other setups may be added here, hydrated from the same orchestrator
+	presets.DoMain(m, presets.NewSimpleInterop(&SimpleInterop))
+}
+
+// TestReorgUnsafeHead starts an interop chain with an op-test-sequencer, which takes control over sequencing the L2 chain and introduces a reorg on the unsafe head
+func TestReorgUnsafeHead(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	ctx := t.Ctx()
+
+	sys := SimpleInterop(t)
+	l := sys.Log
+
+	ia := sys.Sequencer.IndividualAPI(sys.L2ChainA.ChainID())
+
+	l.Info("Stopping batcher")
+	err := sys.L2BatcherA.Escape().ActivityAPI().StopBatcher(ctx)
```

Comment:
There are quite a few calls to `Escape()`, I think if we embed the `stack` types within the `dsl` types, tests would be more readable. DSL is nice, but I'd expect tests to also often use the actual service APIs.